VisualRust/RustGoToDefinitionCommandHandler.cs

```diff
@@ -59,7 +59,7 @@ protected override bool Execute(VSConstants.VSStd97CmdID command, uint options,
             }
 
             // TODO: Taken from AugmentCompletionSession
-            using (var tmpFile = new TemporaryFile(snapshot.GetText()))
+            using (var tmpFile = new TemporaryFile(snapshot.GetText(), System.IO.Path.GetDirectoryName(_buffer.GetFilePath())))
```

Comment:
Compiler is returning an error `The name '_buffer' does not exist in the current context`